src/main/scala/com.snowplowanalytics.iglu/ctl/Storage.scala

```diff
@@ -25,37 +25,107 @@ case class Storage[F[_]](xa: Transactor[F]) {
     * Fetch columns of the given database table
     */
   def getColumns(tableName: String, tableSchema: String)(implicit F: Bracket[F, Throwable]): F[List[Storage.Column]] = {
-    (fr"SELECT column_name FROM information_schema.columns" ++
-      fr"WHERE" ++
-      fr"table_name = $tableName" ++
-      fr"AND table_schema = $tableSchema" ++
-      fr"ORDER BY ordinal_position").query[Storage.Column].to[List].transact(xa)
+    queryInformationSchema(tableName, tableSchema)
```

Comment:
This would look nicer with implicits.  i.e. moving this `map` to the `queryInformationSchema`.